package.json

```diff
@@ -39,7 +41,9 @@
     "react-redux": "^7.2.2",
     "redux": "^4.0.5",
     "redux-persist": "^6.0.0",
-    "redux-saga": "^1.1.3"
+    "redux-saga": "^1.1.3",
+    "redux-thunk": "^2.3.0",
+    "save": "^2.4.0"
```

Comment:
What is this? Where is it used?

Reply:
I think I accidentally ran `npm install redux-thunk - save` instead of `--save` or something, removing